authorization.md

```diff
@@ -33,40 +33,60 @@ bed and room usage and displays statistics on a wall monitor.
 
 ## Registering a SMART Backend Service (communicating public keys)
 
-Before a SMART backend service can run against an EHR, the service must be
-registered with that EHR's authorization service.  SMART does not specify a
+Before a SMART backend service can run against an EHR, the service SHALL
+obtain a client-to-client digital certificate and SHALL
+register with that EHR's authorization service.  SMART does not specify a
 standards-based registration process, but we encourage EHR implementers to
-consider the [OAuth 2.0 Dynamic Client Registration
-Protocol](https://tools.ietf.org/html/draft-ietf-oauth-dyn-reg) for an
-out-of-the-box solution.
-
-No matter how a backend service registers with an EHR's authorization service, a 
-backend service should communicate its **public key** to the SMART EHR using a
-[JSON Web Key Set (JWKS)](https://tools.ietf.org/html/rfc7517) by one of the
-following techniques:
-
-  1. JWKS URL (preferred). This URL communicates the TLS-protected endpoint where the service's
-  public JSON Web Keys can be found. When provided, this URL will match the `jku` header
-  parameter in the service's Authorization JWTs. An advantage of this approach is that
-  it allows a client to rotate its own keys by updating the hosted content at the JWKS URL.
-  2. JWKS directly (allowed, not preferred). If a backend service cannot host a JWKS at a
-  TLS-protected URL, it may supply a JWKS directly to the EHR at registration time. A limitation
-  of this approach is that it does not enable the client to rotate its keys in-band.
+consider using the [OAuth 2.0 Dynamic Client Registration
+Protocol](https://tools.ietf.org/html/draft-ietf-oauth-dyn-reg).
+
+No matter how a backend service registers with an EHR's authorization service, the 
+backend service SHALL register its **client_id** and the **public key** the
```

Comment:
@bakerdb can you update to indicate that the **client_id** is assigned by the server (i.e., a client doesn't specify it)?

```suggestion
backend service SHALL register the **public key** that the
```